client/attest.go

```diff
@@ -19,9 +20,9 @@ const (
 )
 
 // TEEDevice is an interface to add an attestation report from a TEE technology's
-// attestation driver.
+// attestation driver or quote provider.
 type TEEDevice interface {
```

Comment:
Ideally this interface should be renamed to TEEQuoteProvider but not updating it to keep changes to a minimum.